packages/volto/src/components/manage/Blocks/Teaser/schema.js

```diff
@@ -127,3 +127,15 @@ export const gridTeaserDisableStylingSchema = ({ schema, formData, intl }) => {
   schema.fieldsets = schema.fieldsets.filter((item) => item.id !== 'styling');
   return schema;
 };
+
+export const gridTeaserDisableAlignHandlersSchema = ({
+  schema,
+  formData,
+  intl,
+}) => {
+  schema.properties.styles.schema.fieldsets[0].fields =
+    schema.properties.styles.schema.fieldsets[0].fields.filter(
+      (item) => !['align'].includes(item),
+    );
+  return schema;
+};
```

Comment:
@sneridagh Why is this here, but not used?

Reply:
Do you think this should be only in VLT?